test/openAPI/features/db_modify.feature

```diff
@@ -1,19 +1,24 @@
+@method=POST @endpoint=/database/modify
 Feature: API endpoint that allows users to create or modify the Digital Registries database schema.
-  Request endpoint: POST /database/modify/
 
-  Background:
-    Given User wants to create or modify the Digital Registries database schema
+  @smoke
+  Scenario: The user successfully creates the Digital Registries database schema smoke test type
 
-  Scenario: The user successfully creates the Digital Registries database schema
-    Given The requested database schema does not exist in the database
-    When The user sends a valid request to create a database schema
-    Then The operation to create a new database schema completes successfully
+    Given User wants to create or modify the Digital Registries database schema
+    When User sends POST request with given Information-Mediator-Client header
```

Comment:
@ingmarvali i have 2 questions. 

1. What is going to happen when we send request without payload, just with information-mediator-client header. Is it going to return us status 400 - Bad request ? 
2. What if information-mediator-client header is invalid, what should we expect to get.

Reply:
@KarolinaKopacz answers from Ingmar: 

```
1. Yes.
2. We should ask Aleksander Reitsakas
the IM is needed for Information Mediator.
```

Reply:
I will just add body to the request but I will not now add status 400 - Bad request , because there is not enough information about this response